week-5-serving-seldon/k8s/seldon-iris.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: iris-model
+  namespace: default
+spec:
+  name: iris
+  predictors:
+  - graph:
+      implementation: SKLEARN_SERVER
+      modelUri: gs://seldon-models/v1.15.0-dev/sklearn/iris
+      name: classifier
+    name: default
+    replicas: 1
```

Comment:
why do you need this file?

Reply:
Just to make sure that seldon and serving works as expected, it's like "hello world" to test initial infrastructure setup.

But in general, indeed, I don't need the file.